pkg/common/util/addr.go

```diff
@@ -0,0 +1,35 @@
+package util
+
+import (
+	"fmt"
+	"net"
+	"path/filepath"
+)
+
+// GetUnixAddr returns a unix address with the designated
+// path. Path is con
```

Comment:
looks like comment is uncompleted?

Reply:
Yes, thanks for catching this. I'll complete the comment.